Core/PaymentMethodFactory.php

```diff
@@ -47,6 +48,8 @@ public static function create($sPaymentMethodType)
                 return new PaypalPaymentMethod();
             case CreditCardPaymentMethod::getName(true):
                 return new CreditCardPaymentMethod();
+            case SepaDirectDebitPaymentMethod::getName(true):
+                return new SepaDirectDebitPaymentMethod();
             case SofortPaymentMethod::getName(true):
                 return new SofortPaymentMethod();
             case SepaCreditTransferPaymentMethod::getName(true):
```

Comment:
I personally would prefer to have the payment methods sorted alphabetically. So, I would suggest to move the `SepaCreditTransferPaymentMethod` above `SepaDirectDebitPaymentMethod`.